### DISCUSSION TITLE
Issue with refreshing .NET MAUI Blazor app on Android device

### DISCUSSION THREAD

**Discussion by hiiro-sips** [dotnet/maui#15462] [General]
Hello,

I'm currently testing app development using .NET MAUI Blazor. I started by building an app using the MAUI Blazor template in Visual Studio 2022, and when I deploy the app directly to my Android device, it launches successfully on the first attempt. However, if I make even a minor change to the text in index.razor and try to debug it again, the screen turns completely blank.

Interestingly, if I uninstall the app from my Android device and then debug it again, it works fine. However, it's quite inefficient to uninstall the app every time. I was wondering if anyone else has encountered a similar issue and if there's a workaround or solution available.

I appreciate any insights or suggestions you may have regarding this issue. Thank you in advance!

**Comment by ghost** [comment]
Same issue here, recently started building new .NET MAUI app, when i use an emulator everything works but when I use a local device I need to uninstall the debug app with every debug or else i get the same blank screen!

@hiiro-sips Did you find a solution to this problem? or workaround?

**Comment by francis900212** [comment]
same issue here

**Comment by bosdimexltd** [comment]
same
